src/PKSim.Presentation/Presenters/Charts/SimulationTimeProfileChartPresenter.cs

```diff
@@ -23,15 +23,17 @@ namespace PKSim.Presentation.Presenters.Charts
    public interface ISimulationTimeProfileChartPresenter : IChartPresenter<SimulationTimeProfileChart>,
       ISimulationAnalysisPresenter,
       IPKAnalysisWithChartPresenter,
+      IListener<RenamedEvent>,
```

Comment:
Added this event here to listen to global rename event

Reply:
What about other profile types (Range Plots in Pop etc.)? Or was Time profile the only one where RenamedEvent was missing?